src/SeaNetMicron.cpp

```diff
@@ -89,43 +98,35 @@ void Micron::configure(const MicronConfig &config,uint32_t timeout)
     //some conversions
     int ad_interval = (((config.resolution/config.speed_of_sound)*2.0)*1e9)/640.0;
     int number_of_bins = config.max_distance/config.resolution;
-    uint16_t left_limit = (((M_PI-config.left_limit.rad)/(M_PI*2.0))*6399.0);
-    uint16_t right_limit = (((M_PI-config.right_limit.rad)/(M_PI*2.0))*6399.0);
+    uint16_t left_limit = (((M_PI+config.left_limit.rad)/(M_PI*2.0))*6399.0);
+    uint16_t right_limit = (((M_PI+config.right_limit.rad)/(M_PI*2.0))*6399.0);
```

Comment:
Yes, it is right. The micron's zero is at the back as the figure below.

![screenshot from 2016-03-29 17 19 18](https://cloud.githubusercontent.com/assets/4711921/14122492/76d12a60-f5d2-11e5-8933-0f3900c5a0bc.png)